src/material/bottom-sheet/bottom-sheet.md

```diff
@@ -50,6 +50,8 @@ export class HobbitSheet {
 ```
 
 ### Configuring bottom sheet content via `entryComponents`
+**This is only necessary if your project is using ViewEngine. It doesn't have to be specified for 
+projects running on Ivy.**
```

Comment:
I would phrase this like
```markdown
You only need to specify `entryComponents` if your project uses ViewEngine. Projects
using Angular Ivy don't need `entryComponents`.
```

Reply:
corrected
Also, I've updated  a portal's docs as @dev054 suggested